FEATURES.md

```diff
@@ -126,6 +126,7 @@ If your PR makes notable changes to Winit's features, please update this section
 * Hidden titlebar
 * Hidden titlebar buttons
 * Full-size content view
+* Forward file-open evnets
```

Comment:
I'm not really sure what the goal of the FEATURES.md file is, but this seems a bit out of place here. There's also a typo.

Reply:
Yeah, I wasn't sure about this either so I'm removing it.